data/publications.yml

```diff
@@ -194,13 +194,13 @@
   url: https://arxiv.org/abs/1907.08982
   date: 2019-07-21
   authors:
-  - name: Jonas Rothfuss
-  - name: Fabio Ferreira
-  - name: Simon Boehm
-  - name: Simon Walther
-  - name: Maxim Ulrich
-  - name: Tamim Asfour
-  - name: Andreas Krause
+    - name: Jonas Rothfuss
```

Comment:
Sorry looks like I indeed missed some despite going through this twice, thanks for catching these!

Reply:
i have auto-format on save :)

Reply:
I believe you mentioned this before, but for some reason the formatter doesn't seem to work for me. I installed the `YAML- Red Hat` from the VS Code extension market, and [added the following to my `settings.json`](https://stackoverflow.com/a/73428061/24021108):
```
"[yaml]": {
        "editor.defaultFormatter": "redhat.vscode-yaml"
    },
```

But it still allows saving incorrectly formatted yaml file without auto-format (only a red mark would be shown):
<img width="512" alt="image" src="https://github.com/user-attachments/assets/7ae8c532-f59e-4890-9776-c94e1028f1a7">

Reply:
i have `"editor.defaultFormatter": "esbenp.prettier-vscode"` in `settings.json`

we could add one of these hooks to `.pre-commit-config.yaml` to automate the formatting

```yml
  - repo: https://github.com/google/yamlfmt
    rev: v0.10.0
    hooks:
      - id: yamlfmt
        args: [-formatter, retain_line_breaks=true]

  - repo: https://github.com/pre-commit/mirrors-prettier
    rev: v4.0.0-alpha.8
    hooks:
      - id: prettier
        args: [--write] # edit files in-place
```

Reply:
Brilliant! It makes sense to install an automatic hook to format these.

For the prettier hooks, [it wouldn't not work and has been archived](https://github.com/pre-commit/mirrors-prettier):
> prettier made some changes that breaks plugins entirely

Also I'm a bit unsure why the `check-yaml ` doesn't seem to do anything?
https://github.com/janosh/awesome-normalizing-flows/blob/a59d33421cc90c651a148cba2f0d2c3d6920d332/.pre-commit-config.yaml#L23-L34

Reply:
`check-yaml` only checks that all YAML are valid and parseable. it doesn't care about formatting.

> For the prettier hooks, [it wouldn't not work and has been archived](https://github.com/pre-commit/mirrors-prettier):

thanks for letting me know! i missed that

Reply:
> thanks for letting me know!

thanks for letting me know too :)